### DISCUSSION TITLE
Withdrawal problem

### DISCUSSION THREAD

**Discussion by viesajalbd** [blockscout/blockscout#10215] [Support]
Hello sir.I have withdraw my redstone ballence before 1 month ago.blockchain over but not recieve my eth

**Comment by viesajalbd** [comment]
![Screenshot_20240608_233942_Chrome](https://github.com/blockscout/blockscout/assets/140193863/76fc87ee-33ca-4fbc-9fd7-8a4976469676)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
